doc/getting_started/getting_started_go.rst

```diff
@@ -245,3 +245,197 @@ To send bare Lua code for execution, use ``Eval``:
 .. code-block:: go
 
     resp, err = connection.Eval("return 4 + 5", []interface{}{})
+
+.. _getting_started-go-comparison:
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Feature comparison
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+There are two more connectors from open-source community available:
+`viciious/go-tarantool <https://github.com/viciious/go-tarantool>`_ and
+`FZambia/tarantool <https://github.com/FZambia/tarantool>`_. 
+
+Last update: December 2021
+
+..  list-table::
+    :header-rows: 1
+    :stub-columns: 1
+
+    *   -
+        -   `tarantool/go-tarantool <https://github.com/tarantool/go-tarantool>`_
+        -   `viciious/go-tarantool <https://github.com/viciious/go-tarantool>`_
+        -   `FZambia/tarantool <https://github.com/FZambia/tarantool>`_
+
+    *   -   License
+        -   BSD 2-Clause
+        -   MIT
+        -   BSD 2-Clause
+
+    *   -   Last update
+        -   2021
+        -   2021
+        -   2021
+
+    *   -   Known Issues
+        -   `#50 <https://github.com/tarantool/go-tarantool/issues/50>`_,
+            `#89 <https://github.com/tarantool/go-tarantool/issues/89>`_,
+            `#94 <https://github.com/tarantool/go-tarantool/issues/94>`_
+        -   `#41 <https://github.com/viciious/go-tarantool/issues/41>`_
+        -   No known issues
+
+    *   -   Documentation
+        -   README with examples, API described in code comments
+        -   README with examples, code comments
+        -   README with examples
+
+    *   -   Testing / CI / CD
+        -   GitHub Actions
+        -   Travis CI
+        -   GitHub Actions
+
+    *   -   Github Stars
+        -   127
+        -   43
+        -   12
+
+    *   -   Static Analysis
+        -   No
+        -   golint
+        -   golangci-lint
+
+    *   -   Packaging
+        -   go get
+        -   go get
+        -   go get
+
+    *   -   Code coverage
+        -   No
+        -   No
+        -   No
+
+    *   -   msgpack driver
+        -   `vmihailenco/msgpack/v2 <https://github.com/vmihailenco/msgpack/tree/v2>`_ (`#124 <https://github.com/tarantool/go-tarantool/issues/124>`_)
+        -   `tinylib/msgp <https://github.com/tinylib/msgp>`_
+        -   `vmihailenco/msgpack/v5 <https://github.com/vmihailenco/msgpack/tree/v5>`_
+
+    *   -   Async work
+        -   Yes
+        -   Yes
+        -   Yes
+
+    *   -   Batching
+        -   No
+        -   No
+        -   No
```

Comment:
There is automatic batching of commands inside connection, but maybe it's not what Batching means here (if not - I think better to explain what batching is).

Reply:
It's about data/tuples batching

Reply:
Is data batching and command batching over connection are different? Maybe you can point me to some docs about this  - I have not used Tarantool a lot.

Reply:
It's about reading big chunk of data (for example, `space:select{}`) with batches, an ability to process *single* heavy request with cursors or something like this. It's not liek some special terminology since there are no such in-built feature in tarantool inself.

Reply:
Hm, if this was called `cursor iteration` or sth like this - could be more clear what's this section about. But nevermind! :)

Reply:
I disagree with the terminology here as well. Chunked transfer of a large response is the separate thing.

When I initially draft the list of features I assumed that a synchronous connector may need ability to collect several requests into a batch and send them at once to lower `(rtt+execution time)*N` wait time to just `rtt + (execution time)*N` (or even near to `rtt + execution time` if the requests are not CPU intensive). An asynchronous connector pipelines requests by construction, so I guess batching is not necessary here (at least not for performance).

It seems, this row requires a clarification.

Reply:
Oh, well, then I rework the section

Reply:
Since all of the connectors are async, do we really need one?

Reply:
I don't mind removing it.

Reply:
Removed